### DISCUSSION TITLE
Can I use a fallback extension for `generateHTML`?

### DISCUSSION THREAD

**Discussion by ghost** [ueberdosis/tiptap#3905] [General Questions]
When I use `generateHTML` in the front-end of my app, the app crashes when `generateHTML` (or `generateText`) tries to resolve a node/mark, that is not in the list of extensions. How can I render the content anyway?

Let’s say I am using the `Bold` or the `BulletList` extension in the Editor itself and on my front-end I want to render the editor content without these extensions, like:

```js
return generateHTML(json, [
        Document,
        Paragraph,
        Text,
        // Fallback
        RenderAnythingElseAsText,
      ]);
```

**Comment by bdbch** [comment]
Hey @JonathanDoelan! Not yet. Feel free to create a feature request for this over here:
https://github.com/ueberdosis/tiptap/discussions/new?category=feature-requests
